rpm/yamui.spec

```diff
@@ -1,6 +1,6 @@
 Name:       yamui
 Summary:    Minimal UI tool for displaying simple graphical indicators
-Version:    1.0.3
+Version:    1.0.6
```

Comment:
Matter of taste and opinion, but I've usually done the version updating after merging just in case something else gets merged first and the number then needs to be updated anyway (i.e. not have version bump included in a pull req).